code/modules/modular_computers/hardware/hard_drive.dm

```diff
@@ -125,16 +143,16 @@
 			return F
 	return null
 
-/obj/item/computer_hardware/hard_drive/Destroy()
-	QDEL_LIST(stored_files)
-	return ..()
-
-/obj/item/computer_hardware/hard_drive/Initialize(mapload)
-	. = ..()
+/// Tries to find the first file by type. Returns null on fail.
+/obj/item/computer_hardware/hard_drive/proc/find_file_by_type(filetype)
```

Comment:
This proc is entirely unused, do we need it still?

Reply:
added this as a convenience function in case someone needed it in the future, should I remove it?

Reply:
I thought it was something that you added to do something then removed their uses. I don't know how it would be useful though. I'll leave it up to you.

Reply:
I wanted to add this function as a helper for the messenger so I can open it without having to pass a string but I realized I can pass the messenger itself, so I left this in as a helper for modular computers. Idk